tlair/src/main/scala/at/forsyte/apalache/tla/lir/io/Printer.scala

```diff
@@ -11,8 +11,11 @@ import at.forsyte.apalache.tla.lir.values._
 abstract class Printer {
   def apply(p_ex: TlaEx): String
 
-  def apply(p_decl: TlaDecl): String = ""
-
+  def apply(p_decl: TlaDecl): String = {
+    // Telling the compiler we know we're not using p_decl here
+    locally(p_decl);
```

Comment:
This is orthogonal to this PR, but it might be cleaner to keep this abstract and override in implementations?

Reply:
It's a good point. I didn't actually dig in to see if there's every a reason to expect this to default to an empty string. Perhaps it should always be overriden?

Reply:
I'd say so, @Kukovec you have thoughts on this?